client/mocks.go

```diff
@@ -3,7 +3,7 @@ package client
 //go:generate mockgen -source=./client.go -destination=./mock.go -package client ClientInterface
 //go:generate mockgen -source=./client.go -destination=./mock.go -package client ClientWithResponsesInterface
 
-import "github.com/golang/mock/gomock"
+import "go.uber.org/mock/gomock"
```

Comment:
I wondered why this change was made:

> Update, June 2023: This repo and tool are no longer maintained. Please see [go.uber.org/mock](https://github.com/uber/mock) for a maintained fork instead.

Was posted on golang.com/golang/mock's GitHub page.